fintech-examples/fintech-ui/proxy-dev.conf.json

```diff
@@ -0,0 +1,11 @@
+{
+  "/fintech-api-proxy/*": {
```

Comment:
we probably need to bind it to nginx environment variable to run it in docker

Reply:
this was done by sascha. It came in due to rebase.